### ISSUE TITLE
Rename "NixOS Weekly" to "NixOS news" (or similar)

### ISSUE THREAD

**Issue by j-steinbach** [NixOS/nixos-weekly#161]
"NixOS Weekly" implies weekly news. It *promises* weekly news. This doesn't happen.

These are the latest issues. There are often weeks, even months in between. 

- 05 Published on 05 Sep 2021
- 04 Published on 07 Jun 2021
- 03  Published on 08 Apr 2021
- 02 Published on 22 Feb 2021
- 01 Published on 20 Jan 2021

Why is this a problem?

- Breaking promises is bad
- It makes NixOS look like abandonware (similar to these "expect more to come soon" posts written in 2013 and followed up by tumbleweed?)
- It shows possible new users that the community doesn't care all that much. "They can't even get their *homepage* right, why should I take a closer look?"

Instead of making a promise and not holding it, why not rename the issues? Possible names:

- Monthly NixOS (12 issues, unlikely)
- Bi-Monthly NixOS (6 issues a year, should be do-able)
- NixOS Quarterly (4 issues, do-able)
- NixOS Updates (overlaps with actual system updates/releases)
- NixOS News (no number of issues implied)
- NixNews

**Comment by Mic92** [comment]
* News from NixOS
* NixOS newsletter

**Comment by Synthetica9** [comment]
It's also not just NixOS news, but about the entire Nix ecosystem. I propose "Nix News", which is also sounds like the Dutch "Niks nieuws" which means "Nothing new [to report]", which I think is funny :P

**Comment by Mic92** [comment]
German is also close: "Nichts neues". I am also for `Nix News`

**Comment by arianvp** [comment]
Nix News is an _amazing_ name.

**Comment by domenkozar** [comment]
Thanks for starting this discussion @j-steinbach!

I was hoping we'd eventually end up on a weekly schedule, but there doesn't seem enough news to get there yet.

I agree that the current name sets the wrong expectation, so I'm not opposed to changing it.

My hesitation is only that it creates quite some work (migration of all infra) and confusion to existing users (needs to be communicated).

I've bought http://nixos.news/, as we've decided to prefer nixos since nix is an overloaded term. Also nix.news is expensive :)

**Comment by j-steinbach** [comment]
I am just happy that I am not the only one being bothered by this :)

It might be a good idea to start with a clean slate and also "implement" https://github.com/NixOS/nixos-weekly/issues/145, while we are at it.

Can we put the "Nix News" vs "NixOS News" up to vote? Or is that a can of worms that shouldn't be opened again? I find this a strong argument: _It's also not just NixOS news, but about the entire Nix ecosystem._ (@Synthetica9)

----

**Goodbye, NixOS Weekly. Hello NixOS News.**

Starting this fall/winter/new year, we are changing the "weekly" issues (oh well, we tried) to a a running release. 

_What does that mean for me?_ 

Absolutely nothing! You will continue to find the latest news and articles about Nix ecosystem at the [same location](https://nixos.org/blog/index.html) you are used to.

_What changes?_

The issues are now called "NixOS News", _D'oh_. Also the numbers don't start over each year, but instead just continue from 0 to infinity and beyond! This was a common point of confusion -- and we heard 'ya! We are always listening to feedback and eager to improve our community. Have something on your mind? Hop on Discourse/Discord/Matrix/IRC or create a new issue/discussion on Github.

**Comment by sternenseemann** [comment]
I doubt that people pay enough attention to the frequency of email newsletters they receive to notice that NixOS Weekly doesn't arrive in the advertised frequency.

**Comment by j-steinbach** [comment]
> I doubt that people pay enough attention to the frequency of email newsletters they receive to notice that NixOS Weekly doesn't arrive in the advertised frequency.

Hey, sorry for not getting back to this sooner, as I didn't find the right words for it. I believe [this](https://news.ycombinator.com/item?id=5002262) discusses it in a good way. 

To sum it up: Yes, you are right, this is indeed something most people won't notice. But some will, and they will be bothered by it. Heck, _I noticed_ and it bothers me: A website is like a business card, you can't just half-ass it. The [NixOS](https://nixos.org/) project literally prides itself on its _reliability_, and then, one click further down the road you stare at some tumble-weeds on the "weekly" newsletter? Really?


In the end we have to ask our self the question: Do we want to provide a quality product? Or is "good enough" good enough?


![2021-10-05_22-53](https://user-images.githubusercontent.com/69524139/136100982-92010218-6829-4e5c-b627-732b707695da.png)


 ---

> My hesitation is only that it creates quite some work (migration of all infra)

@domenkozar Isn't there a template where we can simply change the name from `NixOS Weekly` to `NixOS News` (or `Nix News`)? The corresponding menu entry on the [website](https://nixos.org/blog/index.html) is called "Blog", so that side doesn't need any extra attention.

**Comment by sternenseemann** [comment]
> Do we want to provide a quality product?

I don't want to provide a “product”, this is a volunteer-run open source project after all. I think reliability in terms of code written and newsletter sent out are two different things. Some things never will work out perfectly because volunteer time and energy is limited. We should not hold us to standards of companies with marketing departments or we will burn out people left and right.

(Maybe a bit besides the point of renaming or not, but I think it is still important to say.)

**Comment by j-steinbach** [comment]
A quick search shows me that there are different definitions for "product". I take it as "a thing produced by labor". If I have my hobby garden, I can still produce tomatoes. But you are right -- this goes off-topic, and produces senseless discussions. (It might be that my usage of the world "deliver" was too strong.)

I just try to make my point clear why I started this discussion and why I _strongly_ believe that this should be changed.

---

> We should not hold us to standards of companies with marketing departments or we will burn out people left and right.

I definitely agree, but that doesn't mean we shouldn't have standards.

**Comment by raboof** [comment]
I agree we should be mindful that people are volunteering their time, so we shouldn't be dismissive when the result doesn't meet "standards of companies with marketing departments" (though at the same time, I'm happy to see we regularly meet and exceed "standards of companies" in many aspects ;) ).

This means we should not "create work"/pressure people into doing "quite some work", such as infra migration.

That said, I think calling it "Nix News" would be a great improvement, so when anyone wants to do any work moving in that direction I'd appreciate & applaud that ;). I'm OK with 'NixOS News" as well but don't like it as much. As for the domain, perhaps we could have https://news.nixos.org (like https://weekly.nixos.org)? Slightly weird to have the 'nixos' in there when it's about Nix in general, but that goes for all of nixos.org so that's par for the course ;)

**Comment by domenkozar** [comment]
I think this comes down to if we want to publish news weekly or not.

It makes little sense to change the name if the long term goal is weekly posts.

Weekly posts have always been my long term goal, but at the short-term cost of sloppyness.

Would you rather have weekly or monthly updates?

**Comment by dasJ** [comment]
I think going with monthly releases as long as there's at least 1 item would be a good middle-ground.
If there are items stacking up (at least 1 per week), we could switch to a more frequent schedule.
